tests/system/python/api/test_audit.py

```diff
@@ -79,20 +79,40 @@ def test_default_get_audit(self, fledge_url, wait_time, request_params, total_co
         assert 200 == r.status
         r = r.read().decode()
         jdoc = json.loads(r)
+
+        elems = jdoc['audit']
+        if storage_plugin == 'postgres':
+            for i, o in enumerate(elems):
+                if o['details'] and o['details']['name'] and o['details']['name'] == 'sqlite':
+                    #print("Skipping item " + str(i) + " = " + str(o))
+                    elems.remove(o)
+
+        if storage_plugin == 'postgres':
+            # sqlite has an additional entry: CONAD, INFORMATION, '{{"name":"sqlite", ...}'
+            # so total_count is being decresed in case of 'postgres' storage engine
+            # same for audit_count
+            if total_count > 1:
+                total_count = total_count -1
+
+            if audit_count > 1:
+                audit_count = audit_count -1
```

Comment:
I believe this logic will fail given the query params in API. better option is to skip this and have separate test /test data for test_default_get_audit_postgres  etc. based on storage_plugin fixture

Reply:
@pintomax limit and skip logic will fail. Even this will start failing for sqlite etc too. see my previous comment.